src/cc/libbpf.h

```diff
@@ -89,6 +89,10 @@ int bpf_close_perf_event_fd(int fd);
 
 int bpf_obj_pin(int fd, const char *pathname);
 int bpf_obj_get(const char *pathname);
+int bpf_obj_get_info_by_fd(int prog_fd, void *info, int *info_len);
```

Comment:
Does `_by_fd` need to be here? It is inconsistent with other bpf_xx functions, the fd is generally an implicit parameter.